applications/nrf_desktop/doc/settings_load.rst

```diff
@@ -0,0 +1,29 @@
+.. _settings_load:
+
+Settings load module
+####################
+
+Use the Settings load module to trigger loading the data from non-volatile memory.
+
+Module Events
+*************
+
++----------------+-------------+-------------------+-----------------+------------------+
+| Source Module  | Input Event | This Module       | Output Event    | Sink Module      |
++================+=============+===================+=================+==================+
+|                |             | ``settings_load`` |                 |                  |
++----------------+-------------+-------------------+-----------------+------------------+
+
+Configuration
+*************
+
+The Settings load module is enabled for every nRF Desktop device with :ref:`zephyr:settings` enabled - when :option:`CONFIG_SETTINGS` is set.
+
+Zephyr assumes that the application is responsible for calling the :cpp:func:`settings_load` after having done all necessary initialization.
+The mentioned function is called on the ``settings_load`` module initialization.
+Make sure that all the settings handlers have been registered and the :cpp:func:`bt_enable` was called before the ``settings_load`` module is initialized.
+
+You can set the ``CONFIG_DESKTOP_SETTINGS_LOAD_BY_THREAD_ENABLE`` Kconfig option to load the settings by the separate thread in background instead of use of the system workqueue for that purpose.
```

Comment:
```suggestion
You can set the ``CONFIG_DESKTOP_SETTINGS_LOAD_BY_THREAD_ENABLE`` Kconfig option to load the settings in a separate thread in the background instead of using the system workqueue for that purpose.
```